update the "using EDB Cloud API" doc with latest account management behavior change (#1949)
EnterpriseDB/docs: product_docs/docs/edbcloud/beta/reference/index.mdx

}
```

Store the access token and refresh token in environment variables for future use. For example:

```
RAW_ACCESS_TOKEN="eyJhbGc.......1Qtkaw2fyho"
REFRESH_TOKEN="v1.MTvuZpu.......sbiionEhtTw"
```

!!!note
The access token obtained at this step will be used in the next step to exchange the EDB Cloud token only.
!!!

If not successful, you receive one of the following errors:
- `authorization_pending` - continue polling using the suggested interval retrieved when you [requested the device code](#request-the-device-code-using-curl).

- `slow_down` - slow down and use the suggested interval retrieved when you [requested the device code](#request-the-device-code-using-curl). To avoid receiving this error due to network latency, you should start counting each interval after receipt of the last polling request's response.
- `expired_token` - you have not authorized the device quickly enough, so the `device_code` has expired. Your application should notify you that it has expired and to restore it.
- `access_denied`

## Exchange the EDB Cloud token using `curl`
!!!note
   The `get-token` script executes this step. You don't need to make this call if you are using the script.
!!!

> Get the EDB Cloud token with the raw token got previously:

Reviewer: @xugy99 , did you see this suggestion?

> Get the EDB Cloud token with the raw token got previously:

Author: @drothery-edb yeah I saw it last night, it was quite late (I am located in Beijing) and there 's no button for me to click as the automatically merge so I just address this in the latest commit manually 😄

```
curl -s --request POST \
     --url "https://portal.edbcloud.com/api/v1/auth/token" \
     --header "content-type: application/json" \
     --data "{\"token\":\"$RAW_ACCESS_TOKEN\"}"
```

If successful, the call returns:
- `token` - The bearer token used to access the EDB Cloud API.

For example:
```
{
  "token": "eyJhbGc.......0HFkr_19Vr7w"
}
```

> This token is similar to the previous access token but it is the one that EDB Cloud API recognizes.

Reviewer: ```suggestion
This token, as opposed to the raw access token, is recognized by the EDB Cloud API.
```

}
```

Store the raw access token and refresh token in environment variables for future use. For example:

```
RAW_ACCESS_TOKEN="eyJhbGc.......1Qtkaw2fyho"
REFRESH_TOKEN="v1.MTvuZpu.......sbiionEhtTw"
```

> The token got from this step is still the raw access token, you still need to follow the step of [Exchange for EDB Cloud token](#exchange-the-edbcloud-token-using-curl) for exchanging an EDB Cloud token.

Reviewer: ```suggestion
The token you obtain from this step is the raw access token, you need to exchange this token for an EDB Cloud token. See [Exchange for EDB Cloud token](#exchange-the-edbcloud-token-using-curl) for more information.
```
